service_container.rst

```diff
@@ -534,8 +534,8 @@ You can also fetch parameters directly from the container::
     {
         // ...
 
-        // this ONLY works if you extend the base Controller
-        $adminEmail = $this->container->getParameter('admin_email');
+        // this ONLY works if you extend the base AbstractController
+        $adminEmail = $this->container->get('parameter_bag')->get('admin_email');
 
         // or a shorter way!
         // $adminEmail = $this->getParameter('admin_email');
```

Comment:
we will need to remove this line

Reply:
@xabbuh I made this minor reword while merging: https://github.com/symfony/symfony-docs/commit/eeeed9268a96a998f7e2d793b1e70a223915902f